docs/usage/self-hosted-configuration.md

```diff
@@ -134,6 +134,10 @@ Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer pack
 
 This configuration will be applied after all other environment variables so that it can be used to override defaults.
 
+## dockerCacheVolume
+
+Create and mount volume for manager cache files. Will be removed after repository processing.
```

Comment:
```suggestion
Creates and mounts a temporary Docker volume that is used as a cache for package manager files.
This volume will be removed after repository processing.
```

How about something like this?

Please check if my description is technically correct. 😉

Reply:
Thanks a lot!